plugin/evm/atomic/sync/atomic_sync_summary.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package sync
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ava-labs/avalanchego/codec"
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/coreth/plugin/evm/message"
+
+	"github.com/ava-labs/avalanchego/snow/engine/snowman/block"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/crypto"
+)
+
+var _ message.Syncable = (*AtomicSyncSummary)(nil)
+
+// Codec is the codec manager that contains the codec for AtomicSyncSummary and
+// other message types that are used in the network protocol. This is to ensure that the codec
+// version is consistent across all message types and includes the codec for AtomicSyncSummary.
+var Codec codec.Manager
+
+func init() {
+	var err error
+	Codec, err = message.NewCodec(AtomicSyncSummary{})
+	if err != nil {
+		panic(fmt.Errorf("failed to create codec manager: %w", err))
+	}
+}
+
+// AtomicSyncSummary provides the information necessary to sync a node starting
+// at the given block.
+type AtomicSyncSummary struct {
+	*message.BlockSyncSummary `serialize:"true"`
+	AtomicRoot                common.Hash `serialize:"true"`
+
+	summaryID  ids.ID
+	bytes      []byte
+	acceptImpl message.AcceptImplFn
```

Comment:
Is there a reason why we need to duplicate these fields from `message.BlockSyncSummary`? Since we're embedding `*message.BlockSyncSummary` already, it seems like it may be unnecessary.

We can likely also get rid of `GetBlockHash`, `GetBlockRoot`, etc, since they are already present in `BlockSyncSummary` and embedded here, right?

Reply:
These are unexported fields, the bytes and summaryID here should include `AtomicRoot`, so we cannot use them from blockSyncSummary at best we can use another strut but feels like unnecessary. 

removed the methods